change/@ni-nimble-components-af276551-d3c5-47e9-8241-5a435257aa31.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Could you update the HLD where we describe the `data` property (and our rationale for initially choosing to use a property instead of a method)?